projects/cli-files/go-cat/cmd/root.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+)
+
+func Execute() {
+	path, err := checkArg()
+	if err != nil {
+		fmt.Println(err)
```

Comment:
How useful a message will this show to the user? Will it be obvious to them what's wrong and what they should do about it?

(Also, should this go to stdout or stderr?)